code/scripting/api/libs/ui.cpp

```diff
@@ -240,6 +241,19 @@ ADE_FUNC(playCutscene, l_UserInterface, "string Filename, boolean RestartMusic,
 	return ADE_RETURN_NIL;
 }
 
+ADE_FUNC(linkTexture, l_UserInterface, "texture texture", "Links a texture directly to librocket.", "string", "The url string for librocket, or an empty string if invalid.")
+{
+	texture_h* tex;
+
+	if (!ade_get_args(L, "o", l_Texture.GetPtr(&tex)))
+		return ade_set_error(L, "s", "");
+
+	if(!tex->isValid())
```

Comment:
Check that `tex` is nullptr here.  I discovered recently that if an argument is present but has the wrong type, `ade_get_args` will succeed despite the pointer not being assigned.